force-app/main/default/classes/Homework.cls

```diff
@@ -63,38 +82,51 @@ public with sharing class Homework {
     {
         // Declare two Integers, with the values 6 and 7, in a single 
         // statement.
+        Integer firstValue = 6, secondValue = 7;
     }
 
     public void booleanFromString()
     {
         // Create a boolean whose value is initialised from the text
         // ‘true’.
+        
+        Boolean myBooleanValue = true;
```

Comment:
`true` is a Boolean literal, not text. You need to enclose it in single quotes and use the appropriate method to convert it from a String.